### ISSUE TITLE
Outdated depencency puppetlabs/concat blocks upgrading

### ISSUE THREAD

**Issue by llauren** [broadinstitute/puppet-certs#30]
Hi!

The Certs module requires a puppetlabs/concat module of version less than 3. Any chance of bumping this up to a current concat (now 4.2.1)? 

I could force-upgrade, but that feels rather brutish... and anything i break is then mine to fix :)

**Problem:**
https://github.com/broadinstitute/puppet-certs/blob/92544b7d8480e1c26a95bb7fc59ee8ea8ea19f62/metadata.json#L65

**Suggested fix:**
```
    { "name":"puppetlabs/concat","version_requirement":">= 1.1.1 < 5.0.0" }
```

Thanks,
 ~rL

**Comment by rcalixte** [comment]
@llauren thanks for the notice, this will get merged into a new release soon

**Comment by rcalixte** [comment]
@llauren this should be fixed in 2.2.1. Thanks for the issue!
